x-pack/plugins/observability_solution/apm/public/components/app/service_inventory/index.tsx

```diff
@@ -281,6 +283,15 @@ export function ServiceInventory() {
     });
   }, [mainStatisticsStatus, mainStatisticsData.items, setScreenContext]);
 
+  useEffect(() => {
+    if (
+      mainStatisticsStatus === FETCH_STATUS.SUCCESS &&
+      comparisonFetch.status === FETCH_STATUS.SUCCESS
+    ) {
+      onPageReady();
```

Comment:
Its good to call the `performance.clearMeasures("name")` in the return callback of the useEffect to clear any entries that was captured on the previous render.

Reply:
I clear the measure in the context https://github.com/elastic/kibana/pull/180309/commits/8858e37c66fab861ae62719a5aa4a99f1343fe3e